enterprise/app/services/messages/audio_transcription_service.rb

```diff
@@ -46,8 +50,6 @@ def transcribe_audio
 
     temp_file_path = fetch_audio_file
 
-    response_text = nil
-
     File.open(temp_file_path, 'rb') do |file|
```

Comment:
```suggestion
    transcribed_text = nil

    File.open(temp_file_path, 'rb') do |file|
```

We need a default value to start with, just like previously implmented